packages/@o3r/testing/src/core/playwright/component-fixture.ts

```diff
@@ -26,11 +26,31 @@ export class O3rComponentFixture<V extends O3rElement = O3rElement> implements C
     this.rootElement = rootElement;
   }
 
-  protected throwOnUndefined<T extends O3rElement>(elemnt?: T) {
-    if (!elemnt) {
+  /** @deprecate not supported for Playwright */
+  protected throwOnUndefined<T extends O3rElement>(element?: T): T;
+  /** @inheritdoc */
+  protected async throwOnUndefined<T extends O3rElement>(element: Promise<T | undefined>): Promise<T>;
+  /**
+   * @param element
+   * @deprecate not supported for Playwright
```

Comment:
you provide an implementation but the way you phrase the deprecated it does not seem possible to implement it which is a bit confusing.
Do you mean that we deprecate it because it goes against playwright's locator design?

Reply:
If I put something like `@deprecated not valid in case of Playwright due to the Locator implementation`, what do you think>?